web/src/utils/context/userContext.tsx

```diff
@@ -37,33 +40,36 @@ const UserProvider: React.FC<Props> = ({ children }) => {
 
     // await Auth.currentSession();
 
-    const me = await getMe();
+    const me = await getMe().then((user) => {
+      if (user != null) {
+        setUser(user.user);
+        setIsAdmin(user.isAdmin);
+      }
+    });
     console.debug({ msg: "getMe response", me });
-    if (me != null) {
-      setUser(me.user);
-      setIsAdmin(me.isAdmin);
-    }
   }
 
   const amplifyListener: HubCallback = async ({ payload: { event, data } }) => {
     console.debug("aws-amplify Hub recieved event", { event, data });
     switch (event) {
       case "signIn":
+        setamplifyLoggedIn(true);
+        setamplifyUser(data);
       case "cognitoHostedUI":
-        await getUser().then(() => {
+        await getUser().then((out) => {
           if (user !== undefined) {
-            setLoading(false);
+            //setLoading(false );
```

Comment:
is this commented out?
Seems like it would still be needed if the call to getMe fails?

Reply:
It called the same thing in the useEffect, so was effectively not needed. Can just be deleted I think

Reply:
ok